packages/react/src/context/slash-id-context.tsx

```diff
@@ -239,21 +208,21 @@ export const SlashIDProvider: React.FC<SlashIDProviderProps> = ({
     }
   }, [oid, baseApiUrl, sdkUrl, state, tokenStorage, analyticsEnabled]);
 
-  useEffect(() => {
-    if (state !== "initial") {
-      const slashId = sidRef.current!;
-      // @ts-expect-error TODO expose the type
-      const getUserFromEvent = ({ token }) => {
-        const userFromToken = new User(token, sidRef.current!);
-        if (!user || userFromToken.token !== user.token) {
-          setUser(userFromToken);
-        }
-      };
-      slashId.subscribe("idFlowSucceeded", getUserFromEvent);
-
-      return () => slashId.unsubscribe("idFlowSucceeded", getUserFromEvent);
-    }
-  }, [state, user]);
+  // useEffect(() => {
```

Comment:
Let's try deleting this, we have it in git anyway if it needs to be resurrected

Reply:
Was leaving it here to get your thoughts! I will remove it